README.md

```diff
@@ -6,34 +6,80 @@
 
 At the core of Dagger.jl is a scheduler heavily inspired by [Dask](http://dask.pydata.org/en/latest/). It can run computations represented as [directed-acyclic-graphs](https://en.wikipedia.org/wiki/Directed_acyclic_graph) (DAGs) efficiently on many Julia worker processes.
 
-# DAG creation interface
 
-Here is an example DAG:
+## Installation
+
+You can install it by typing
 
 ```julia
-using Dagger
+julia> ] add Dagger
+```
 
-p = delayed(f; options...)(42)
-q = delayed(g)(p)
-r = delayed(h)(53)
-s = delayed(combine)(p, q, r)
+## Usage
+
+Once installed, the `Dagger` package can by used by typing
+
+```julia
+using Dagger
 ```
-The connections between nodes `p`, `q`, `r` and `s` is represented by this dependency graph:
 
-![graph](https://user-images.githubusercontent.com/25916/26920104-7b9b5fa4-4c55-11e7-97fb-fe5b9e73cae6.png)
+The main function for using Dagger is *delayed*
 
 `delayed(f; options...)`
 
-Returns a function which when called creates a `Thunk` object representing a call to function `f` with the given arguments. If it is called with other thunks as input, then they form a graph with input nodes directed at the output. The function `f` get the result of evaluating the input thunks.
+When called creates a `Thunk` object representing a call to function `f` with the given arguments. If it is called with other thunks as input, then they form a graph with input nodes directed at the output. The function `f` get the result of the input Thunks.
+Thunks don't pass keyword argument to the function `f`.
 
-To compute and fetch the result of a thunk (say `s`), you can call `collect(s)`. `collect` will fetch the result of the computation to the master process. Alternatively, if you want to compute but not fetch the result you can call `compute` on the thunk. This will return a `Chunk` object which references the result. If you pass in a `Chunk` objects as an input to a delayed function, then the function will get executed with the value of the `Chunk` -- this evaluation will likely happen where the input chunks are, to reduce communication.
- 
-Options to `delayed` are:
+
+Options kwargs... to `delayed` are passed to the scheduler to control its behavior:
```

Comment:
Hmm, ok I see that you actually put this down here... I would prefer that this goes inline with "Thunks don't pass keyword argument to the function `f`.".